web/docs/testing.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
After you merge the main branch, I think you should see a `guides` folder, I would suggest we keep all the guides there. I didn't migrate anything yet there, but we can start putting new things there.